conf/routes

```diff
@@ -6,6 +6,8 @@
 # An example controller showing a sample home page
 GET     /                           controllers.HomeController.index
 GET     /dashboard                  controllers.DashboardController.dashboard
+GET     /seriesDetails              controllers.SeriesDetailsController.seriesDetails
+POST    /seriesDetails/submit       controllers.SeriesDetailsController.seriesSubmit()
```

Comment:
Not a big deal, but why do we have parentheses in the POST route but not the GET one?